docs/csharp/quick-starts/hello-world.yml

```diff
@@ -0,0 +1,171 @@
+    Console.WriteLine(firstName);
+    firstName = "Maria";
+    Console.WriteLine(firstName);
+    ```
+
+    Notice that the same line of code prints two different messages, based on
+    the value stored in the `firstName` variable.
+
+    You may have also noticed the "Hello" was missing in the last two messages. Let's fix
+    that now.  Modify the last line to the following:
+
+    ```csharp
+    Console.WriteLine("Hello " + firstName);
+    ```
+
+    Click 'Run' again to see the results.
+- title: Work with strings
+  durationInMinutes: 4
+  content: |
+    Your last edit was our first look at what you can do with strings. Let's explore more.
+
+    The `+`, when applied to strings, creates a new string with the text from the first followed by the text from the second. This is referred to as *string concatenation*. You can combine any number of strings, including string variables.  Try this:
+
+    ```csharp
+    string firstName = "Maria";
+    string aFriend = "Sage";
+    Console.WriteLine("My friends are " + firstName + " and " + aFriend);
+    ```
+
+    As you explore more with strings, you can find the length of a string using `Length`.  Add the following code at the bottom of the interactive window:
+
+    ```csharp
+    Console.WriteLine("The name " + firstName + " has " + firstName.Length + " letters");
+    Console.WriteLine("The name " + aFriend + " has " + aFriend.Length + " letters");
+    ```
+
+    > [!TIP]
+    > This is a good time to explore on your own. You've learned that `Console.WriteLine()`
+    > writes text to the screen. You've learned how to declare variables and
+    > concatenate strings together. Experiment in the interactive window. Write
+    > some more messages yourself. Build strings using the `+` operator.
+- title: Do more with strings
+  durationInMinutes: 4
+  content: |
+    Another useful method is Trim, and related methods TrimStart and TrimEnd. These methods remove leading and trailing spaces:
+
+    ```csharp
+    string message = "      Hello World!       ";
+    Console.WriteLine("[" + message + "]");
+
+    string trimmedMessage = message.TrimStart();
+    Console.WriteLine("[" + trimmedMessage + "]");
+
+    trimmedMessage = message.TrimEnd();
+    Console.WriteLine("[" + trimmedMessage + "]");
+
+    trimmedMessage = message.Trim();
+    Console.WriteLine("[" + trimmedMessage + "]");
+    ```
+
+    This sample reinforces a couple important concepts for working with strings. The methods that manipulate strings return new string objects rather than making modifications in place. You can see that each call to any of the Trim methods returns a new string, but does not change the original message.
+
+    Other methods perform more general edits on a string. You've probably used a search and replace command in an editor or word processor. The Replace method substitutes a replacement string for all occurences of a sub-string in a string. Try it for yourself. Add this code above:
```

Comment:
typo: occurences

I don't think substring needs hyphen

the explanation of Replace is a bit confusing. Maybe introduce the concept of parameters here? Could say something like "The Replace method searches for all occurrences of the first substring and replaces them with the second substring in a string"?

Reply:
Or link to here: https://docs.microsoft.com/en-us/dotnet/api/system.string.replace?view=netcore-2.0#System_String_Replace_System_String_System_String_  😄